### PR TITLE
feature：logging config loaded source

### PR DESCRIPTION
**Pull request by oslijunw**
for :#63

### CODE REVIEW

**Comment by oslijunw** [comment]
I don't know how to fix the lint to pass the first test, I haven't changed much in the code, so can't figure out how to solve this problem with the ci process

File: pydantic_settings/main.py [hunk 1]
```diff
@@ -16,6 +18,7 @@
     SecretsSettingsSource,
 )
 
+log = logging.getLogger('pydantic')
```
Comment (Kludex) [inline pydantic_settings/main.py:21]: Maybe `pydantic.settings`? I'm not sure what's the best name here.

Comment (oslijunw) [reply pydantic_settings/main.py:21]: tks for your reply, I will change that

**Comment by hramezani** [comment]
Close as per comment [here](https://github.com/pydantic/pydantic-settings/issues/63#issuecomment-1577441967)

**Pull request closed without merging**